src/Compilers/CSharp/Portable/Binder/Binder_Deconstruct.cs

```diff
@@ -705,6 +704,17 @@ private static void FlattenDeconstructVariables(ArrayBuilder<DeconstructionVaria
                         }
                         return new DeconstructionVariable(builder, node);
                     }
+                case SyntaxKind.IdentifierName:
+                    {
+                        var identifier = (IdentifierNameSyntax)node;
```

Comment:
You may have been looking at my first commit only. I pushed two commits. The second one does a lookup, although I'm not sure it uses all the right options.